techniques/systemSettings/security/fileAlterationMonitoring/2.0/metadata.xml

```diff
@@ -14,7 +14,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<TECHNIQUE name="Monitor a file or directory content">
+<TECHNIQUE name="File and directory monitoring">
```

Comment:
This is close but not very evocative for those who know and deal with these concepts. How about "File integrity"?